src/test/java/hudson/plugins/active_directory/docker/TheFlintstonesTest.java

```diff
@@ -101,6 +115,56 @@ public void simpleLoginFails() throws Exception {
         }
     }
 
+    @Issue("JENKINS-45576")
+    @Test
+    public void loadGroupFromGroupname() throws Exception {
+        String groupname = "The Rubbles";
+        GroupDetails group = j.jenkins.getSecurityRealm().loadGroupByGroupname(groupname);
+        assertThat(group.getName(), is("The Rubbles"));
+    }
+
+    @Issue("JENKINS-45576")
+    @Test
+    public void loadGroupFromAlias() throws Exception {
+
```

Comment:
Would you mind to delete this empty line?

Reply:
done